README.rst

```diff
@@ -136,11 +136,23 @@ The official documentation is hosted on ReadTheDocs at http://larray.readthedocs
 Get in touch
 ============
 
-- Report bugs, suggest features or view the source code `on GitHub`_.
-- For questions, ideas or general discussion, use the `mailing list`_.
+- To be informed of each new release, please subscribe to this `mailing list`_.
+- For questions, ideas or general discussion, please go to our `Google Users Group`_.
+- To report bugs, suggest features or view the source code `on GitHub`_.
```

Comment:
s/code on GitHub/code, go to GitHub/ or "code, use GitHub", or something... otherwise the end of the sentence does not work with the start of the sentence.